blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/PreaggregateConversions.java

```diff
@@ -129,10 +130,12 @@
         List<PreaggregatedMetric> list = new ArrayList<PreaggregatedMetric>(enums.size());
         for (BluefloodEnum en : enums) {
             Locator locator = Locator.createLocatorFromPathComponents(tenant, en.getName().split(NAME_DELIMITER, -1));
-            BluefloodEnumRollup rollup = new BluefloodEnumRollup();
-            rollup = rollup.withEnumValue(en.getValue(), 1L);
-            PreaggregatedMetric metric = new PreaggregatedMetric(timestamp, locator, DEFAULT_TTL, rollup);
-            list.add(metric);
+            if (!ExcessEnumReader.getInstance().isInExcessEnumMetrics(locator)) {
+                BluefloodEnumRollup rollup = new BluefloodEnumRollup();
+                rollup = rollup.withEnumValue(en.getValue(), 1L);
+                PreaggregatedMetric metric = new PreaggregatedMetric(timestamp, locator, DEFAULT_TTL, rollup);
+                list.add(metric);
+            }
```

Comment:
We haven't yet defined how we are going to expose those to ELE.  I don't think they want us to return errors.  We'll need to talk to them about it.  I've added a jira story to track it.  In the mean time I'll add a log mesg.